src/app/legacy/psammead-assets/src/svgs/navigationIcons.jsx

```diff
@@ -4,7 +4,7 @@ import styled from '@emotion/styled';
 const svgSize = '2.75rem'; // 44px
 
 const MenuIcon = styled.svg`
-  color: #fff;
+  color: #000000;
```

Comment:
Should this be imported from psammead-styles too?

Reply:
I guess it could be? I followed the pattern of what was there before it as it was hex instead of the psammead colour for white

Reply:
I would agree w/ @nicola-d-bbc good spot! Now that `psammead-styles` lives in legacy we should import from it as the Nav is still based on the legacy Nav :)

Reply:
yep makes sense, will change it now 👍